docs/code-snippets/outlook-snippets.yaml

```diff
@@ -1023,6 +1324,29 @@ Office.MessageCompose.getSelectedDataAsync:
     function setCallback(asyncResult) {
         // check for errors
     }
+Office.MessageCompose.internetHeaders:
+  - |-
+    Office.context.mailbox.item.internetHeaders.getAsync(["header1", "header2"], callback); 
+
+    function callback(asyncResult) { 
+        var dictionary = asyncResult.value; 
+        var header1_value = dictionary["header1"]; 
```

Comment:
`header1_value` -> `headerValue`

Reply:
I think this one makes sense. If the user wanted header2, then they could say "header2_value".